hikari/__init__.py

```diff
@@ -64,6 +64,7 @@
 from hikari.applications import TeamMembershipState
 from hikari.applications import TokenType
 from hikari.audit_logs import *
+from hikari.auto_mod import *
```

Comment:
I would personally rather `hikari.automod`. I'm ok with `auto_mod` if you rather it

Reply:
myself (alternate thommo) and thommo agree with dav on this one

Reply:
`automod` would clash with everywhere else in this pr where its written as `auto_mod` or `AutoMod`

Reply:
My comment was to change the whole module into that name, not only here :P

Reply:
> My comment was to change the whole module into that name, not only here :P

This effects other modules as well

Reply:
Also just to give my opinion on this while python's style guide does the whole "merge words together if the name is short enough" with stuff like `gettext` I don't ever use that in my own code since it gets a bit inconsistent when you use that name as a part of other longer names as well